tests/models/test_main.py

```diff
@@ -102,22 +102,20 @@ def test_user_with_invalid_email_domain_cannot_be_changed_to_admin_role(self, ro
         assert str(exc.value) == 'invalid_admin_domain'
 
 
-def test_framework_should_not_accept_invalid_status():
-    app = create_app('test')
-    with app.app_context(), pytest.raises(ValidationError):
-        f = Framework(
-            name='foo',
-            slug='foo',
-            framework='g-cloud',
-            status='invalid',
-        )
-        db.session.add(f)
-        db.session.commit()
+class TestFrameworks(BaseApplicationTest):
```

Comment:
Move these into a new class inheriting from `BaseApplicationTest` so the context is created for us.